src/fx-data-download.py

```diff
@@ -9,6 +9,7 @@
 import time
 import urllib.request
 import socket
+from datetime import datetime
```

Comment:
We're importing already `datetime` (few lines above), why we need to import it the 2nd time?